examples/raytracing-rgen/out9.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
Comment here since GitHub won't let me comment directly on the rgen.spv binary:

This example appears incomplete. I tried running:
spirv-run -t out.yaml rgen.spv

and I got an exception: "Cannot print value!". I got the same error when trying to run:
spirv-run -t out.json rgen.spv

You should support printing of template variables. This is a very important feature of SPIRV-Interpreter which differentiates it from other similar projects.